curve25519-dalek/Cargo.toml

```diff
@@ -67,6 +68,7 @@ alloc = ["zeroize?/alloc"]
 precomputed-tables = []
 legacy_compatibility = []
 group = ["dep:group", "rand_core"]
+group-bits = ["group", "ff/bits"]
```

Comment:
FWIW the @RustCrypto crates just call this `bits`. I find `group-bits` a bit weird because the `bits` part is only accidentally dependent on `group` because that's how we import `ff`, but I guess it does get across that it activates the `group` feature too.

Reply:
My preference is scoping it since group itself is optional, whereas RustCrypto crates are traditionally built around group. If `bits` is the way forward, please feel free to say so (as I'm inclined to do nothing when a decision on naming is ambiguous as it is right now).

Reply:
Just a suggestion, I don't care strongly either way

Reply:
@pinkforest for a third opinion? ^_^

Reply:
`ff-bits` :)